.github/workflows/main.yml

```diff
@@ -64,10 +64,15 @@ jobs:
           sudo apt-get install -y --allow-unauthenticated libopenblas-dev liblapack-dev g++ libensmallen-dev libhdf5-dev libarmadillo-dev libcurl4-openssl-dev
           wget https://github.com/USCiLab/cereal/archive/refs/tags/v1.3.2.tar.gz
           tar -xvzpf v1.3.2.tar.gz
+          #
           # These directives cause warnings on CRAN:
           # https://github.com/USCiLab/cereal/blob/master/include/cereal/external/base64.hpp#L28-L31
           # The command below comments them out.
           sed -i 's|#pragma|// #pragma|' cereal-1.3.2/include/cereal/external/base64.hpp
+          #
+          # Fix cereal compilation on clang 19+; see
+          # https://github.com/USCiLab/cereal/pull/835
+          sed -i 's|::template apply|::apply|' cereal-1.3.2/include/cereal/types/tuple.hpp
```

Comment:
Would be interesting to have the patch at cereal as well

Reply:
Yeah, hopefully they will merge it there soon enough. :+1: